src/dmd/frontend.d

```diff
@@ -47,17 +48,63 @@ immutable struct Diagnostics
     }
 }
 
+/// Indicates the checking state of various contracts.
+enum ContractChecking : CHECKENABLE
+{
+    /// Initial value
+    default_ = CHECKENABLE._default,
+
+    /// Never do checking
+    disabled = CHECKENABLE.off,
+
+    /// Always do checking
+    enabled = CHECKENABLE.on,
+
+    /// Only do checking in `@safe` functions
+    enabledInSafe = CHECKENABLE.safeonly
+}
+
+static assert(
+    __traits(allMembers, ContractChecking).length ==
+    __traits(allMembers, CHECKENABLE).length
+);
```

Comment:
Unittest

Reply:
Should I create a unit test to make sure the enums are the same?

Reply:
Done.